controllers/v1/profile.js

```diff
@@ -0,0 +1,55 @@
+const axios = require('axios')
+let profileHelper = require(MODULES_BASE_PATH + '/profile/helper')
+
+module.exports = class profile {
+	static get name() {
+		return 'profile'
+	}
+
+	async read(req) {
+		return new Promise(async (resolve, reject) => {
+			try {
+				// Define headers for the user service request
+				let headers = {
+					internal_access_token: 'internal_access_token',
+					'Content-Type': 'application/json',
+				}
+				// Fetch user details from user service
+				const userResponse = await axios.get(
```

Comment:
@MallanagoudaB this is entirely wrong . you can re-use the function in generics/services/entity-management of projects service. see how it is currently used